lib/globals.js

```diff
@@ -12,6 +13,11 @@ async function setup(jestConfig = {}) {
     didAlreadyRunInWatchMode = true;
   }
 
+  onExit(async () => {
+    await teardownDynamoDbLocalDocker();
+    process.exit();
```

Comment:
Did you find that the process didn't exit without this `process.exit()`? There's nothing that should be interrupting the exit as far as I can tell. We should be able to remove this line.

Reply:
Confirmed, works without `process.exit()`. I wasn't actually sure which "process" is being handled and figured it was safer to quit than not. Also, it was in the [example](https://github.com/facebook/jest/issues/6800#issuecomment-442037011) I copied :D